light_curves/lc_classifier.ipynb

```diff
@@ -0,0 +1,983 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#ask sktime if it likes the data type of X\n",
+    "from sktime.datatypes import check_is_mtype\n",
+    "\n",
+    "check_is_mtype(X_train, mtype=\"pd-multiindex\", scitype=\"Panel\", return_metadata=True)\n",
+    "#check_is_mtype(X_test, mtype=\"pd-multiindex\", scitype=\"Panel\", return_metadata=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4. Run Machine Learning Algorithms on the light curves\n",
+    "\n",
+    "We choose to use [sktime](https://www.sktime.net/en/stable/index.html) algorithms beacuse it is a library of many algorithms specifically tailored to time series datasets.  It is based on the sklearn library so syntax is familiar to many users.\n",
+    "\n",
+    "Types of classifiers are listed [here](https://www.sktime.net/en/stable/api_reference/classification.html).\n",
+    "\n",
+    "Our method is to do a cursory check of a bunch of classifiers and then later drill down deeper on anything with good initial results.  We choose to run a loop over ~9 classifiers that seem promising and check the accuracy scores for each one.  Any classifier with a promising accuracy score could then be followed up with detailed hyperparameter tuning, or potentially with considering other classifiers in that same type."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true,
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "#what is the list of all possible classifiers that work with multivariate data\n",
+    "#all_tags(estimator_types = 'classifier')\n",
+    "classifiers = all_estimators(\"classifier\", filter_tags={'capability:multivariate':True})\n",
+    "classifiers\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#pick one from each type of classifiers\n",
+    "from sktime.classification.kernel_based import Arsenal \n",
+    "from sktime.classification.interval_based import CanonicalIntervalForest\n",
+    "from sktime.classification.hybrid import HIVECOTEV2\n",
+    "from sktime.classification.kernel_based import RocketClassifier\n",
+    "from sktime.classification.deep_learning import CNNClassifier\n",
+    "from sktime.classification.ensemble import WeightedEnsembleClassifier\n",
+    "from sktime.classification.dictionary_based import IndividualTDE\n",
+    "from sktime.classification.distance_based import KNeighborsTimeSeriesClassifier\n",
+    "from sktime.classification.feature_based import RandomIntervalClassifier\n",
+    "from sktime.classification.feature_based import Catch22Classifier\n",
+    "from sktime.classification.shapelet_based import ShapeletTransformClassifier\n",
+    "from sktime.classification.dummy import DummyClassifier\n",
+    "\n",
+    "names = [\"Arsenal\",                     #kernel based\n",
```

Comment:
Would be cool to see one or two examples with plots separately (the ones which takes less time) and then leave the loop with all possible classifiers for the end of the notebook for enthusiasts who can wait

Reply:
done.